### PR TITLE
Removed code for deleteExtraTags

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Removed code for deleteExtraTags where it was causing garbage collection.

Issue #16

### CODE REVIEW

File: cip.go [hunk 1]
```diff
@@ -46,18 +46,14 @@ func main() {
 		"delete-extra-tags",
 		false,
 		"delete tags in the destination registry that are not declared"+
-			" in the Manifest (default: false); implies -garbage-collect")
+			" in the Manifest (default: false);")
```
Comment (listx) [inline cip.go]: Please also delete the trailing semicolon

**Comment by listx** [comment]
Also please reword your commit text like this:

```
deleteExtraTags: turn off garbage collection
```

See https://chris.beams.io/posts/git-commit/ for additional guidance.

**Comment by hpandeycodeit** [comment]
@listx I removed the semicolon but somehow rewording the first commit isn't working. Let me know if you want me to create a new PR?

**Comment by listx** [comment]
You can just `git rebase -i HEAD~4` and squash your 4 commits for this PR into 1 commit, then force-push your branch (e.g. `git push -f origin issue_16`). This PR will auto-update itself.

**Comment by hpandeycodeit** [comment]
@listx I just did it. Let me know if it looks good to you. Thank you!

**Comment by listx** [comment]
Awesome, thank you!

**Comment by hpandeycodeit** [comment]
@listx Thank you!

**Pull request merged**